src/plugins/discover/public/components/discover_grid_flyout/use_flyout_actions.tsx

```diff
@@ -45,95 +28,31 @@ export const useFlyoutActions = ({ actions, ...props }: UseFlyoutActionsParams)
+      onClick: onOpenSingleDoc,
+      label: i18n.translate('discover.grid.tableRow.viewSingleDocumentLinkLabel', {
+        defaultMessage: 'View single document',
+      }),
     },
     {
       id: 'surroundingDocument',
       enabled: Boolean(!viewSurroundingDocument.disabled && dataView.isTimeBased() && dataView.id),
-      Content: () => <SurroundingDocuments onClick={onOpenContextView} href={contextViewHref} />,
+      dataTestSubj: 'docTableRowAction',
+      iconType: 'documents',
+      href: contextViewHref,
+      onClick: onOpenContextView,
+      label: i18n.translate('discover.grid.tableRow.viewSurroundingDocumentsLinkLabel', {
+        defaultMessage: 'View surrounding documents',
+      }),
     },
     ...customActions,
   ];
 
-  const hasEnabledActions = flyoutActions.some((action) => action.enabled);
-
-  if (hasEnabledActions) {
-    flyoutActions.unshift(staticViewDocumentItem);
-  }
-
-  return { flyoutActions, hasEnabledActions };
-};
-
-const ViewDocument = () => {
-  return (
-    <EuiHideFor sizes={['xs', 's', 'm']}>
-      <EuiFlexItem grow={false}>
-        <EuiText size="s">
-          <strong>
-            {i18n.translate('discover.grid.tableRow.viewText', {
-              defaultMessage: 'View:',
-            })}
-          </strong>
-        </EuiText>
-      </EuiFlexItem>
-    </EuiHideFor>
-  );
-};
-
-const SingleDocument = (props: FlyoutActionProps) => {
-  return (
-    <EuiFlexItem grow={false}>
-      <EuiButtonEmpty
-        size="s"
-        iconSize="s"
-        iconType="document"
-        flush="left"
-        data-test-subj="docTableRowAction"
-        {...props}
-      >
-        {i18n.translate('discover.grid.tableRow.viewSingleDocumentLinkTextSimple', {
-          defaultMessage: 'Single document',
-        })}
-      </EuiButtonEmpty>
-    </EuiFlexItem>
-  );
-};
-
-const SurroundingDocuments = (props: FlyoutActionProps) => {
-  return (
-    <EuiFlexGroup alignItems="center" responsive={false} gutterSize="none">
-      <EuiFlexItem grow={false}>
-        <EuiButtonEmpty
-          size="s"
-          iconSize="s"
-          iconType="documents"
-          flush="left"
-          data-test-subj="docTableRowAction"
-          {...props}
-        >
-          {i18n.translate('discover.grid.tableRow.viewSurroundingDocumentsLinkTextSimple', {
-            defaultMessage: 'Surrounding documents',
-          })}
-        </EuiButtonEmpty>
-      </EuiFlexItem>
-      <EuiFlexItem grow={false}>
-        <EuiIconTip
-          content={i18n.translate('discover.grid.tableRow.viewSurroundingDocumentsHover', {
-            defaultMessage:
-              'Inspect documents that occurred before and after this document. Only pinned filters remain active in the Surrounding documents view.',
-          })}
-          type="questionInCircle"
-          color="subdued"
-          position="right"
-          iconProps={{
-            className: 'eui-alignTop',
-          }}
-        />
```

Comment:
@andreadelrio Should we add the help icon back?

Reply:
I don't have the context for why this tooltip was added. @davismcphee how do you envision `helpText` prop working?

Reply:
No worries, figured it was worth a shot to ask. I was thinking the `helpText` prop would allow passing help text that would show the info icon next to the button, but I'm now realizing that might look out of place with the icon-only buttons. I'd be happy to merge this as is if no one feels strongly about the help icon, and maybe we could consider another way to present the information to users as a followup.

cc @kertal in case you have thoughts or context around the help icon.

Reply:
@davismcphee @andreadelrio 

We could add it to tooltips:


<img width="500" alt="Screenshot 2024-01-10 at 10 54 09" src="https://github.com/elastic/kibana/assets/1415710/7a45842f-a207-403e-8efb-74e2246eb69d">
<img width="500" alt="Screenshot 2024-01-10 at 10 53 59" src="https://github.com/elastic/kibana/assets/1415710/bee513af-9def-47e8-b306-b547bd7d5610">

Reply:
Good idea, I hadn't thought of that 👍 This works for me if it seems all good to @andreadelrio on the design side.

Reply:
Looking good, thx!

Reply:
I don't love this from a design point of view for the icon-only version but since this is an optional prop I think it should be ok.